### PR TITLE
Update readme with example picture & golang port reference

### LINKED ISSUE #46: GoLang port of your library
Today I found myself looking for a GoLang library that can do what your JS one does - converting circles into polygons. I couldn't find anything, so I wrote this: https://github.com/chrusty/go-circle-to-polygon. I've completely copied your math, and am very grateful for your work!

This is not an "issue" as such - just wanted to let you know, and to say thanks.

### PR DESCRIPTION
**Pull request by johachi**
## Description

- Add an example image
- Update the example to be code that creates the polygon shown in the example image.
  This is also the same parameters used by the [go/golang port](https://github.com/chrusty/go-circle-to-polygon/blob/main/README.md)
- This closes #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 The GeoJSON spec does not support circles. If you wish to create an area that represents a circle, your best bet is to create a polygon that roughly approximates the circle. In the limit of the number of edges becoming infinite, your Polygon will match a circle.
 
+`circleToPolygon([173.283966, -41.270634], 20000, { numberOfEdges: 32 })` would yield the polygon below:
+
+![Example of a polygon with 20000 meter radius, 32 edges and center in 173.283966,-41.270634 (lon,lat)](example.png "Example circle (20km radius with 32 edges)")
```
Comment (johachi) [inline README.md:7]: The image is not visable in preview since it links to an image that has not yet been merged to master.

**Review by gabzim — APPROVED** [review]
LGTM 👍

**Comment by gabzim** [comment]
Something we could add @johachi is an `.npmignore` file to make sure the image does not become a part of the npm package in future versions. That way we don't increase the size of the package in case somebody's using this on the front end.

**Comment by johachi** [comment]
> Something we could add @johachi is an `.npmignore` file to make sure the image does not become a part of the npm package in future versions. That way we don't increase the size of the package in case somebody's using this on the front end.

@gabzim , I didn't think about that. I added one now and included the image and all test files (`*specs.js`). Looks good?

**Comment by gabzim** [comment]
Looks good! Merge away!

**Pull request merged**